api/app/src/TestHelpers/DeputyTestHelper.php

```diff
@@ -5,18 +5,16 @@
 namespace App\TestHelpers;
```

Comment:
The Fixtures class has a createDeputy() method you could use instead of adding a new test helper.